package/MDAnalysis/analysis/hole2/templates.py

```diff
@@ -0,0 +1,97 @@
+
```

Comment:
add boring boilerplate file header (comments/licence...)

Reply:
Updated `# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
` to `# Copyright (c) 2006-2020 The MDAnalysis Development Team and contributors
` -- should this be done everywhere?